src/app/modules/evaluate/component/evaluate-options/evaluate-options.component.ts

```diff
@@ -109,4 +122,8 @@ export class EvaluateOptionsComponent implements OnInit {
   public onResetClick(): void {
     this.resetTrigger.next();
   }
+
+  public getIndexedText() {
```

Comment:
What is this doing exactly? removing all digits?

Reply:
yea so, maybe we can add more localization/optimization, but there is a set of localization keys they are set to day/days etc. Then he created an Enum that used the values 1day/3days. I wanted to utilize his localization.

I attempted to change the enum to match the localization keys, but that broke the compatibility with the existing enum structure in the settings. If we want to create a function that fixes the back porting instead that is fine with me.